generated/nifake/nifake/session.py

```diff
@@ -1295,7 +1317,7 @@ def import_attribute_configuration_buffer(self, configuration):
         '''
         vi_ctype = _visatype.ViSession(self._vi)  # case S110
         size_in_bytes_ctype = _visatype.ViInt32(0 if configuration is None else len(configuration))  # case S160
-        configuration_ctype = get_ctypes_pointer_for_buffer(value=_converters.convert_import_buffer_to_array(configuration), library_type=_visatype.ViInt8)  # case B520
+        configuration_ctype = get_ctypes_pointer_for_buffer(value=_converters.convert_import_buffer_to_array(configuration, _visatype.ViInt8), library_type=_visatype.ViInt8)  # case B520
```

Comment:
Getting rid of `library_type` seems like a step back. Is this because of that `unused` parameter?
Maybe we should rethink this a bit.

Reply:
Yes, the `unused` parameter results in `library_type` to be passed into the converter.
The other options I considered:
1. Not passing in `library_type` to converters. This would require us to implement N versions (one for each supported number type) of `convert_timedeltas_to_seconds`, `convert_timedeltas_to_milliseconds`, and `convert_timedeltas_to_microseconds` each.
2. Add a new case in code generator for something like `python_api_two_parameter_converter_name`, which goes against the desire to keep code generator as simple as possible.

Another thing to keep in mind is the case of using the converter for a single `timedelta`, where the converter takes two parameters - `time` and `library_type`. It would be nice to be consistent with it.

Reply:
Can we keep the `library_type` parameter, and instead of `unused` assert that it's what we expect? Or does that complicate things too much?

Reply:
Added an assert.